Docs: sync visualization docs with source (pt2) (#126813)
grafana/grafana: docs/sources/visualizations/panels-visualizations/visualizations/geomap/index.md

- [Route](#route-layer) render data points as a route.
- [Photos](#photos-layer) renders a photo at each data point.
- [Network](#network-layer) visualizes a network graph from the data.

> - [Open Street Map](#open-street-map-layer) adds a map from a collaborative free geographic world database.

Author: ```suggestion
- [OpenStreetMap](#openstreetmap-layer) adds a map from a collaborative free geographic world database.
```

> - [CARTO basemap](#carto-basemap-layer) adds a layer from CARTO Raster basemaps.

Author: Will add to the dictionary

| Option | Description |
| ------ | ----------- |
| Data | Configure the data settings for the layer. For more information, refer to [Data](#data). |
| Location | Configure the data settings for the layer. For more information, refer to [Location Mode](#location-mode). |
| Size | Configures the size of the markers. The default is `Fixed size`, which makes all marker sizes the same regardless of the data; however, there is also an option to size the markers based on data corresponding to a selected field. `Min` and `Max` marker sizes have to be set such that the markers can scale within this range. |

> | Symbol | Allows you to choose the symbol, icon, or graphic to aid in providing additional visual context to your data. Choose from assets that are included with Grafana such as simple symbols or the Unicon library. You can also specify a URL containing an image asset. The image must be a scalable vector graphic (SVG). |

Author: Will add to the dictionary

| Text label | Configures a text label for each edge. |
<!-- prettier-ignore-end -->

> #### Open Street Map layer

Author: ```suggestion
#### OpenStreetMap layer
```

A map from a collaborative free geographic world database.

> {{< figure src="/static/img/docs/geomap-panel/geomap-osm-9-1-0.png" max-width="600px" alt="Geomap panel Open Street Map" >}}

Author: ```suggestion
{{< figure src="/static/img/docs/geomap-panel/geomap-osm-9-1-0.png" max-width="600px" alt="Geomap panel OpenStreetMap" >}}
```

- **Opacity** from 0 (transparent) to 1 (opaque)
- **Display tooltip** - allows you to toggle tooltips for the layer.

> [About Open Street Map](https://www.openstreetmap.org/about)

Author: ```suggestion
[About OpenStreetMap](https://www.openstreetmap.org/about)
```

{{< figure src="/static/img/docs/geomap-panel/geomap-xyz-9-1-0.png" max-width="600px" alt="Geomap panel xyz example" >}}

- **URL template** - Set a valid tile server URL, with {z}/{x}/{y} for example: `https://tile.openstreetmap.org/{z}/{x}/{y}.png`. Dashboard variables are supported, for example, `https://example.com/maps/${version}/{z}/{x}/{y}.png`.
- **Attribution** sets the reference string for the layer if displayed in [map controls](#show-attribution). Dashboard variables are supported.
- **Min zoom** - Minimum zoom level. Tiles aren't loaded below this level.
- **Max zoom** - Maximum zoom level provided by the server. Beyond this level, the map is rendered by scaling up existing tiles.
- **Opacity** from 0 (transparent) to 1 (opaque)

##### More information

- [Tiled Web Map Wikipedia](https://en.wikipedia.org/wiki/Tiled_web_map)

> - [List of Open Street Map Tile Servers](https://wiki.openstreetmap.org/wiki/Tile_servers)

Author: ```suggestion
- [List of OpenStreetMap Tile Servers](https://wiki.openstreetmap.org/wiki/Tile_servers)
```

#### MapLibre Style layer

The MapLibre Style Layer is a map defined using a MapLibre/Mapbox `style.json` URL. The style contains the URL to the tiles, layer definitions, and more. Typically, they're based on vector tiles as opposed to raster tiles.

- **URL template** - Set a valid style URL. For example: `https://demotiles.maplibre.org/style.json`

> - **Public access token** - An API token for mapbox maps. Only works for `mapbox://` URLs. Refer to [mapbox access tokens documentation](https://docs.mapbox.com/help/dive-deeper/access-tokens/) for more information. In other cases, you might have to include the token in the URL. For example: `https://example.com/map/style.json?key=XXX`.

Author: Will add to dictionary

> - **Public access token** - An API token for mapbox maps. Only works for `mapbox://` URLs. Refer to [mapbox access tokens documentation](https://docs.mapbox.com/help/dive-deeper/access-tokens/) for more information. In other cases, you might have to include the token in the URL. For example: `https://example.com/map/style.json?key=XXX`.

Author: Will add to dictionary

There are five basemap layer types to choose from in a geomap.

> - [Open Street Map](#open-street-map-layer) adds a map from a collaborative free geographic world database.

Author: ```suggestion
- [OpenStreetMap](#openstreetmap-layer) adds a map from a collaborative free geographic world database.
```

> - [CARTO basemap](#carto-basemap-layer) adds a layer from CARTO Raster basemaps.

Author: Will add to dictionary

default_baselayer_config = `{
  "type": "xyz",
  "config": {

>     "attribution": "Openstreetmap",

Reviewer: ```suggestion
    "attribution": "OpenStreetMap",
```

See https://github.com/grafana/grafana/pull/123824

>     "attribution": "Open street map",

Author: ```suggestion
    "attribution": "Openstreetmap",
```
